pf-gensdk.cmd

```diff
@@ -0,0 +1,8 @@
+set SCRIPT_PATH=%~dp0
+set PF_ROOT=%SCRIPT_PATH%..\..\..
+
+call %SCRIPT_PATH%\pf-backupandclean.cmd
+pushd .
+cd %PF_ROOT%\SDKGenerator\
+call ts-node generate.js -destpath %PF_ROOT%\sdks\XPlatCSdk -apiSpecGitUrl -buildIdentifier XPlatCSdk_manual
+popd
```

Comment:
should this script not also do a 'pf-copy-from-generated.cmd' before regenerating?

Reply:
you don't always want that.  you'll overwrite the work you did in the templates.  it depends on where you made the edits 

---
In reply to: [631948623](https://github.com/jplafonta/XPlatCTemplate/pull/16#discussion_r631948623) [](ancestors = 631948623)